packages/components/organisms/f-footer/src/tenants/en-GB.js

```diff
@@ -242,6 +242,13 @@ export default {
                     rel: 'noopener',
                     gtm: 'click_about_modern_slavery_statement'
                 },
+                {
+                    url: '/CmsAssets/media/Files/Common/legal/gender-pay-gap/gender-pay-gap-2019-2021',
```

Comment:
Is the casing correct?  `Files`?

Reply:
This is the link that legal have given us and the link works. Since the deadline is Monday, I don't think we have time to spend too long on clarifying with them